gensim/similarities/nmslib.py

```diff
@@ -10,20 +10,57 @@
 This module contains integration Nmslib with :class:`~gensim.models.word2vec.Word2Vec`,
 :class:`~gensim.models.doc2vec.Doc2Vec`, :class:`~gensim.models.fasttext.FastText` and
 :class:`~gensim.models.keyedvectors.KeyedVectors`.
-
+An instance of NmslibIndexer needs to be created in order to use Nmslib in gensim.
```

Comment:
Please avoid passive language where possible. It's a poor choice in English (personally, I avoid it in general regardless of language), because it's verbose and ambiguous. Consider the alternative:

> To use nmslib, instantiate a :class:`~gensim.similarities.nmslib.NmslibIndexer` class and pass the instance as the ``indexer`` parameter to your model's ``most_similar`` method (e.g. :py:func:`~gensim.models.doc2vec.most_similar`).

So how it effectively says the same thing, but directly, tersely, and unambiguously?

Reply:
I just realized I can push to your branch, so I've applied the above change myself.

Reply:
Thanks for your change!